### DISCUSSION TITLE
Standup (Tuesday, May 3, 2022)

### DISCUSSION THREAD

**Discussion by paotg123** [CarlosCRG19/ijalti-peers#37] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
 ```

**Comment by paotg123** [comment]
**What did you achieve in the last 24 hours?**:
 - I started a course to learn about js, html, jxs React

**What are your priorities for the next 24 hours?**:
 - finish the course

**Blockers**:
 -Puras trabas y errores

**Shoutouts**:
 - @CarlosCRG19 @CristianCazares  @antonieto @BrandonMagana @DavBot02

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - Finish Relational Model for our feature DB #36 with @BrandonMagana 
 - Get introduced to entities in TypeORM

**Blockers**:
 - None

**Shoutouts**:
 - @BrandonMagana for being such a great teammate

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  @antonieto and I have been working on ERM diagrams, and defined the current architecture for the backend

**What are your priorities for the next 24 hours?**:
 -  Finish #36  with @antonieto 
 - Start implementing our data entities using TypeORM

**Blockers**:
- None

**Shoutouts**:
 - @antonieto for being working hard
 - @CarlosCRG19  for helping me solving some git issues

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 -  Started a PERN stack course
 - Finished #31 with @paotg123 and @CristianCazares 

**What are your priorities for the next 24 hours?**:
 - Continue course
 - Start basic mockup of one of our use cases

**Blockers**:
 -

**Shoutouts**:
 - @CristianCazares @paotg123 for working hard 💪

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished a TODO App as an intro to PERN.
 - Define the basics of #31 
 - Research on react hooks

**What are your priorities for the next 24 hours?**:
 - To finish #31 by defining the folder structure

**Blockers**:
 - Still figuring out how react hooks work and useState/useEffect

**Shoutouts**:
 - @paotg123 for starting to learn React
